regression/pages/lms/checkout_page.py

```diff
@@ -0,0 +1,58 @@
+"""
+Payment page.
+"""
+from bok_choy.page_object import PageObject
+
+
+class PaymentPage(PageObject):
+    """
+    Payment page
+    """
+    url = 'https://testsecureacceptance.cybersource.com/checkout'
+
+    def is_browser_on_page(self):
+        return 'checkout' in self.browser.title.lower()
```

Comment:
Using the page title will commonly result in flaky tests. Could you use a page object / css path instead?